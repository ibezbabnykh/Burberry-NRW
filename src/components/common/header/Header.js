import React from 'react';
import { Link } from 'react-router-dom';

import './Header.scss';

const Header = () => {
  return (
    <header className="header">
      <div className="center-holder">
        <Link to="/" className="logo">TVMaze</Link>
      </div>
    </header>
  );
}

export default Header;