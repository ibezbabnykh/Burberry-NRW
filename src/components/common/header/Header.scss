@import 'assets/style/variables.scss';

.header {
    background-color: $color-bg-header;
    padding: 10px 0;

    .logo {
        color: $color-brand;
        font-size: 24px;
        font-weight: 700;
        transition: color 0.3s ease-in-out;

        &:hover {
            color: $color-brand-hover;
        }
    }
}