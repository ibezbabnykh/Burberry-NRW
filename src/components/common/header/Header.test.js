import Header from './Header';

describe('Header component', () => {
  it('should render', () => {
    const result = toJson(shallow(<Header />));
    expect(result).toMatchSnapshot();
  });
});