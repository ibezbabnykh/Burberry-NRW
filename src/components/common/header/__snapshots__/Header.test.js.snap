// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render 1`] = `
<header
  className="header"
>
  <div
    className="center-holder"
  >
    <Link
      className="logo"
      to="/"
    >
      TVMaze
    </Link>
  </div>
</header>
`;
